### DISCUSSION TITLE
Should `IES TM-30-18` be renamed to `IES TM-30-20`?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1168] [Q&A] [answered]
Pretty sure [they didn't change anything](https://luminusdevices.zendesk.com/hc/en-us/articles/4421188803469-What-are-the-TM-30-18-Color-Rendition-Guidelines-Reports-#:~:text=TM%2D30%2D18%20is%20now,%2D30%2D18%20report%20format.) in the actual report for IES TM-30-20.  If true, then you could simply change the names in the codebase to -20 to keep current...?
- J

**Comment by blue-j** [comment]
If anyone could use the spectra for the CES 99 patches, they have it at [https://spectralcalculator.pnnl.gov/calculator](https://spectralcalculator.pnnl.gov/calculator) and attached here just in case someone could use it.  - J

[CES.xlsx](https://github.com/colour-science/colour/files/11711394/CES.xlsx)

**Comment by thomasmansencal** [comment, marked as answer]
Good question, I have been meaning to drop the year entirely and put a method argument that uses it for when it is required in a future update. I did not bother changing it yet because 2018 and 2020 are the same for practical purposes.

**Reply by blue-j** [reply]
I'll just edit the source code for now I think. Thank you!!
